test/unit/api/wide/load/arithmetic/load.hpp

```diff
@@ -26,83 +26,236 @@ auto data_block()
 
-TTS_CASE_TPL("Check ctor from aligned pointer for wide", EVE_TYPE )
-{
-  auto constexpr algt = eve::alignment_v<T>;
-  auto [data,idx]     = data_block<EVE_VALUE, eve::fixed<EVE_CARDINAL>>();
-  auto* ref_ptr       = &data[idx];
+      TTS_EQUAL((eve::load(ref_ptr, eve::lane<EVE_CARDINAL>)) , ref);
+    }
 
-  T ref;
+    TTS_AND_THEN("we load from unaligned pointers to constant")
+    {
+      TTS_EQUAL(T(ref_const_ptr)                                    , ref);
 
-  for(std::size_t i=0;i<EVE_CARDINAL;++i)
-    ref.set(i, data[idx+i]);
+      if constexpr(EVE_CARDINAL == eve::expected_cardinal_v<EVE_VALUE>)
+        TTS_EQUAL(eve::load(ref_const_ptr)                            , ref);
 
-  T constructed(eve::as_aligned<algt>(ref_ptr));
-  TTS_EQUAL(constructed, ref);
+      TTS_EQUAL((eve::load(ref_const_ptr, eve::lane<EVE_CARDINAL>)) , ref);
+    }
 
-  T loaded = eve::load(eve::as_aligned<algt>(ref_ptr), eve::as_<T>());
-  TTS_EQUAL(loaded, ref);
-}
+    TTS_AND_THEN("we load from aligned pointers")
+    {
+      TTS_EQUAL(T(eve::as_aligned<algt>(ref_ptr))                                    , ref);
 
-TTS_CASE_TPL("Check ctor from const aligned pointer for wide", EVE_TYPE )
-{
-  auto constexpr algt = eve::alignment_v<T>;
-  auto [data,idx]     = data_block<EVE_VALUE, eve::fixed<EVE_CARDINAL>>();
-  auto const* ref_ptr = &data[idx];
+      if constexpr(EVE_CARDINAL == eve::expected_cardinal_v<EVE_VALUE>)
+        TTS_EQUAL(eve::load(eve::as_aligned<algt>(ref_ptr))                          , ref);
 
-  T ref;
+      TTS_EQUAL((eve::load(eve::as_aligned<algt>(ref_ptr), eve::lane<EVE_CARDINAL>)) , ref);
+    }
 
-  for(std::size_t i=0;i<EVE_CARDINAL;++i)
-    ref.set(i, data[idx+i]);
+    TTS_AND_THEN("we load from aligned pointers to constant")
+    {
+      TTS_EQUAL(T(eve::as_aligned<algt>(ref_const_ptr))                                    , ref);
 
-  T constructed(eve::as_aligned<algt>(ref_ptr));
-  TTS_EQUAL(constructed, ref);
+      if constexpr(EVE_CARDINAL == eve::expected_cardinal_v<EVE_VALUE>)
+        TTS_EQUAL(eve::load(eve::as_aligned<algt>(ref_const_ptr))                          , ref);
 
-  T loaded = eve::load(eve::as_aligned<algt>(ref_ptr), eve::as_<T>());
-  TTS_EQUAL(loaded, ref);
+      TTS_EQUAL((eve::load(eve::as_aligned<algt>(ref_const_ptr), eve::lane<EVE_CARDINAL>)) , ref);
+    }
+  }
+}
+
+TTS_CASE_TPL("Check conditional load from unaligned pointer for wide", EVE_TYPE)
+{
+  constexpr std::ptrdiff_t algt = eve::alignment_v<T>;
+  auto [data,idx] = data_block<EVE_VALUE, eve::fixed<EVE_CARDINAL>>();
+  auto* ref_ptr             = &data[idx];
+  auto const* ref_const_ptr = &data[idx];
+
+  TTS_WHEN("For some given relative masks")
+  {
+    // Conditional selectors
+    auto il = eve::ignore_last(EVE_CARDINAL/4);
+    auto i1 = eve::ignore_first(EVE_CARDINAL/4);
+    auto kf = eve::keep_first(EVE_CARDINAL/4);
+    auto kl = eve::keep_last(EVE_CARDINAL/4);
+    auto kb = eve::keep_between ( std::min(0L,EVE_CARDINAL)
```

Comment:
This min doesn't make sense

Reply:
will replace it with a proper sub-space of 0->cardinal